docs/docs/upgrading/migrate_v4_v5.md

```diff
@@ -159,28 +159,36 @@ interchain-security-pd q provider params -o json
 
 ### Governance proposals
 
-Submitting the following legacy proposals is still supported:
```

Comment:
Not that familiar but why are we modifying this file (migrating v4 to v5)? Wouldn't the legacy proposals be deprecated in a v6 > v5?

Reply:
the part which was confusing is that the filename does not match with the content as the file contains also migration steps from v2->v3. Added a new file with migration steps for v5->v6